dotcom-rendering/src/lib/sendTargetingParams.ts

```diff
@@ -0,0 +1,17 @@
+import { isArray } from 'lodash';
+import type { EditionCommercialProperties } from '../types/commercial';
+
+export const getTargetingParams = (
+	editionCommercialProperties: EditionCommercialProperties,
+): Map<string, string> => {
+	return new Map(
+		editionCommercialProperties.adTargeting
+			.filter((adTargetingParam) => adTargetingParam.name !== 'sh')
```

Comment:
why do we filter this one out? maybe we could add a comment?

Reply:
I removed it because I checked a few articles in apps rendering and `sh` was not in the targeting-params. Some examples:

<img width="941" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/19683595/e8a87761-868a-4586-a7ab-b077f8c2a366">

<img width="937" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/19683595/470944ab-debb-4264-bf97-df561d02859a">

Makes sense and I will add a comment!